package.json

```diff
@@ -258,11 +256,6 @@
     "deep-equal-in-any-order": "^1.0.10",
     "enzyme": "^3.8.0",
     "enzyme-adapter-react-16": "^1.12.1",
-    "eslint": "^3.14.1",
```

Comment:
i think these deletions need to be repped in the lock file in order for it to pass lock file test remotely